### PR TITLE
Like & Delete button

### PR DESCRIPTION
**Pull request by archmagos**
Like & Delete back-end built, delete built into the front-end

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -178,6 +178,8 @@ GEM
     selenium-webdriver (3.11.0)
       childprocess (~> 0.5)
       rubyzip (~> 1.2)
+    shoulda-matchers (3.1.2)
```
Comment (JohnForster) [inline Gemfile.lock]: What are the shoulda-matchers for?

File: app/models/post.rb [hunk 2]
```diff
@@ -5,4 +5,8 @@ class Post < ApplicationRecord
   def poster_name
     User.find(user_id).name if user_id
   end
+
+  def like_id(post_id)
+    Like.select('id').where(post_id: post_id, user_id: 1).first&.id
```
Comment (JohnForster) [inline app/models/post.rb]: user_id is hard coded to 1 at this point. Is that correct?

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -6,8 +6,15 @@
...
+                                  :method => :delete %>
+  <% else %>
+    # Yvonne's front end here
+  <% end %>
 <% end %>
 
 <%= link_to new_post_path do %>
   New post
-<% end %>
+<% end %>
```
Comment (JohnForster) [inline app/views/posts/index.html.erb]: Missing newline at end of file.

File: app/models/post.rb (hunk 2 above)
Comment (archmagos) [reply app/models/post.rb]: Session data folded in. I'll make another commit 👍

**Comment by archmagos** [comment]
Should be ready for a merge folks
@JohnForster @laythq @YvCodeHong @Ben-893

**Comment by YvCodeHong** [comment]
That looks good to me 👍

**Comment by archmagos** [comment]
Changes made, re-commited.

**Comment by JohnForster** [comment]
:shipit:

**Pull request merged**